packages/payment-detection/src/erc20/fee-proxy-contract.ts

```diff
@@ -23,9 +23,11 @@ export abstract class ERC20FeeProxyPaymentDetectorBase<
   TPaymentEventParameters extends PaymentTypes.IERC20FeePaymentEventParameters,
 > extends FeeReferenceBasedDetector<TExtension, TPaymentEventParameters> {
   /**
+   * @param paymentNetworkId
    * @param extension The advanced logic payment network extensions
+   * @param _currencyManager
```

Comment:
```suggestion
   * @param currencyManager
```
not sure why we need these `@param` comments though ^^

Reply:
https://github.com/RequestNetwork/requestNetwork/pull/950/commits/e57b267a1ac0b14e51f44ff95fe90b465a280409